### ISSUE TITLE
Preview not able to view in desktop 

### ISSUE THREAD

**Issue by scienmanas** [scienmanas/TextUtils#3]
Need to check with css

**Comment by AbhiramKrishnaM** [comment]
Can i work on this issue @scienmanas

**Comment by AbhiramKrishnaM** [comment]
Can you elaborate a little bit more ?

**Comment by scienmanas** [comment]
Yeah sure, fork the repo and fix the bug, it's related to tailwind classes, if you enter a good amount of words says > 250 the preview text is hidden under footer.

**Comment by scienmanas** [comment]
I have assigned you to this issue, try to fix it in 1 week

**Comment by scienmanas** [comment]
Also preview text is not wrapped do see that too

**Comment by AbhiramKrishnaM** [comment]
ok
